### ISSUE TITLE
Print solver warning messages to Python console

### ISSUE THREAD

**Issue by tomflexcompute** [flexcompute/tidy3d#1951]
There are some warnings in the logs but they are not printed in the console. One example is the warning about "TE mode specified in the mode source but no TE modes are found". Users think having this kind of messages printed out would be helpful.

**Comment by momchil-flex** [comment]
So currently, if there are warnings found in the log, a warning is printed saying that.

```
             WARNING: Warning messages were found in the solver log. For more   
             information, check 'SimulationData.log' or use                     
             'web.download_log(task_id)'. 
```

The idea behind this was not to have a large output if there are many warning messages in the log. If I understand correctly what you are proposing is instead to display them all?

**Comment by tomflexcompute** [comment]
Yes essentially. It's one of those situations where we can't make everyone happy. Some people prefer to see all the warnings so they know if there are potential issues. Some people prefer to have less warnings so the console/notebook look cleaner. I think we should probably lean towards the former and users can always turn off the warning if they want.

**Comment by tylerflex** [comment]
just checking in here, I wonder if one possible solution is to just modify the code generating the warning above to add the extra logic needed to grab the log (like the warning message says?) and then just display it? or is that overly simplistic? I guess I'm confused why we couldn't just do that

**Comment by momchil-flex** [comment]
I think we can do it, my question above was basically if there are 20 warnings do we want to display them all? This single warning instead tells you something may be off without being too verbose.

**Comment by tylerflex** [comment]
maybe we can use consolidated logger?
